docs/webapi/grabElementBoundingRect.mustache

```diff
@@ -0,0 +1,14 @@
+Grab the width, height, location of given locator.
+Provide `width` or `height`as second param to get your desired prop.
+Resumes test execution, so **should be used inside an async function with `await`** operator.
+
+```js
+const value = await I.grabElementBoundingRect('h3');
+const value = await I.grabElementBoundingRect('h3', 'width');
+```
+
+If multiple elements found returns an array of texts.
```

Comment:
I know that we have this feature for `grabTextFrom` but I think it was a design mistake. 
Please make this return an object, not an array, to make the behavior more predictable.